presentation/feedprofile/src/main/java/com/hilingual/presentation/feedprofile/profile/FeedProfileViewModel.kt

```diff
@@ -73,17 +81,16 @@ internal class FeedProfileViewModel @Inject constructor(
                 )
             }.toImmutableList()
 
-            val likedDiaryUIModels = likedDiariesModel.diaryList.map { likedDiary ->
+            val likedDiaryUIModels = likedDiariesModel?.diaryList?.map { likedDiary ->
                 likedDiary.toState()
-            }.toImmutableList()
+            }?.toImmutableList() ?: emptyList<FeedDiaryUIModel>().toImmutableList()
```

Comment:
p2
이거 아마 뒤에
```suggestion
            }?.toImmutableList() ?: persistentListOf()
```
로 해도 될 거예요! 전 빈 목록 넣어줄 때 이런 식으로 많이 해줬습니다ㅎㅎ